### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by mdclinic** [community/community#33473] [Actions]
welcome
Where do I write this command line?
(git commit --allow-empty -m "Changed repository ownership"; git push)
So I can be sure to transfer repository

**Comment by Hobart2967** [comment]
Which OS are you using?

Use one of your terminal applications :)

**Reply by mdclinic** [reply]
Odoo 13
Odoo Sh

**Comment by mdclinic** [comment]
It is indeed possible to transfer the ownership of the repository linked to your Odoo.sh project.

After performing the transfer, the platform will be updated through the next commit you push. Even an empty commit would do the trick. If in doubt, just use the following command and Odoo.sh will know your repository has a new owner:

git commit --allow-empty -m "Changed repository ownership"; git push


??????

**Comment by airtower-luna** [comment]
Those are Git commands you could run in a local copy of your repository. If you run them, they'll create an empty commit and push it (and any previous commits) to a remote repository (e.g. on GitHub, depending on configuration).

I guess Odoo is something that's supposed to then do something with your repository? Creating an empty commit seems strange though. :sweat_smile:
